src/common/mod.rs

```diff
@@ -36,6 +37,17 @@ pub fn is_training_mode() -> bool {
     true
 }
 
+#[repr(C)]
+// FUN_71013e7be0 sets this up (13.0.1) so look here if more values are needed
+// If you need full size use gdb to look at allocator
+pub struct PauseMenu {
+    padding: [u8; 0xb60],       // Unknown Values
+    pub stale_move_toggle: u32, // Handles if Stale Moves are on, 0 for off, 1 for on
```

Comment:
This is `pub` but unused. Can we actually use this for the stale menu handling that we already do?

Reply:
Yeah you're right it's unused, we technically can but there's no point since we're using the stale menu handling we already do to get this structure. We can private it if you'd like.